### PR TITLE
Enhance async servlet log

### LINKED ISSUE #26485: Servlet 6.1: Clarify the meaning of "write operation" in the Javadoc for ServletOutputStream.isReady() and, in particular, that this does not include ServletOutputStream.close(), AsyncContext.complete() nor any of the AsyncContext.dispatch() methods. Further clarify the Javadoc regarding the serialization of asynchronous dispatches.
`https://github.com/jakartaee/servlet/issues/273`

### PR DESCRIPTION
**Pull request by pmd1nh**
fixes https://github.com/OpenLiberty/open-liberty/issues/26485

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build
        #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat,io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.6.1_fat_toleration.lWAS,com.ibm.ws.webcontainer.servlet.6.0_fat_toleration.lWAS,com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.3.1_fat.2,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat,io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.6.1_fat_toleration.lWAS,com.ibm.ws.webcontainer.servlet.6.0_fat_toleration.lWAS,com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.3.1_fat.2,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
unrelated failure

```
no component 98743 1 2

com.ibm.ws.ui.tool.explore_fat.gui.nd.rw.ws.allresources FAT tests 12 1 0
 com.ibm.ws.ui.tool.explore.fat.gui.nd.rw.allresources.tests.AllServersTest 5 1 0 11 m 26 s 2
  actionBarTest_withWebSocket 1 1 0 8 m 09 s 4

com.ibm.ws.ui.tool.serverConfig_fat.gui.nd.rw FAT tests 18 0 1
 junit.framework.TestSuite 1 0 1 0 s 3
  com.ibm.ws.ui.tool.serverConfig.fat.gui.nd.rw.tests.ServerConfigValidatorTest 1 0 1 0 s 1

io.openliberty.springboot.fat30_fat FAT tests 1 0 1
 com.ibm.ws.springboot.support.fat.FATSuite 1 0 1 0 s 1
  testBlockingIOServlet60 1 0 1 0 s 1
```

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java [hunk 1]
```diff
@@ -584,7 +581,15 @@ else if (_gotWriter == true) {
 // code above LIBERTY only
             }
             catch (Throwable th) {
-                logger.logp(Level.SEVERE, CLASS_NAME,"closeResponseOutput", "Error.while.closing.response.output",th);
+                //Client can close the connection and cause this closing error. Since there is no re-throw backup and the response is winding down,
+                //there is no beneficial other than this random alarm message(s) in console log.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java]: there is no benefit other than this random alarming message(s) in the console log.

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java [hunk 2]
```diff
@@ -584,7 +581,15 @@ else if (_gotWriter == true) {
...
+                //Since 6.1, log this under trace.
+                if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.SEVERE)) {
+                    logger.logp(Level.SEVERE, CLASS_NAME,"closeResponseOutput", "Error.while.closing.response.output",th);
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java:588]: Is this one that we've had cases on etc?

Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java:588]: github issue. The workaround is to suppress it with the LOG property.

**Comment by pmd1nh** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat,io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.6.1_fat_toleration.lWAS,com.ibm.ws.webcontainer.servlet.6.0_fat_toleration.lWAS,com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.3.1_fat.2,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
#libby

**Review by pnicolucci — APPROVED** [review]

**Comment by pmd1nh** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat,io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.6.1_fat_toleration.lWAS,com.ibm.ws.webcontainer.servlet.6.0_fat_toleration.lWAS,com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.3.1_fat.2,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
One unrelated failure

```
no component 98983 0 1
 com.ibm.ws.ui.tool.explore_fat.gui.nd.rw.ws FAT tests 2 0 1
  junit.framework.TestSuite 1 0 1 0 s 2
   com.ibm.ws.ui.tool.explore.fat.gui.nd.rw.FATSuite 1 0 1 0 s 1
```

**Pull request merged**